internal/net/grpc/logger/logger_test.go

```diff
@@ -830,37 +635,18 @@ func Test_logger_Fatal(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           args: nil,
-		       },
-		       fields: fields {
-		           v: 0,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           args: nil,
-		           },
-		           fields: fields {
-		           v: 0,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		// {
+		// 	name: "fatal log the message",
+		// 	args: args{
+		// 		args: []interface{}{"log message"},
+		// 	},
+		// 	fields: fields{
+		// 		v: 0,
+		// 	},
+		// 	afterFunc: func(a args) {
+		// 		_ = recover()
+		// 	},
+		// },
```

Comment:
WIP?

Reply:
fatal function uses `os.Exit` to exit the program after the fatal message is logged. so I comment out the test case.
I have written the reason on line 616

Reply:
For reference, I found the old way to test with `os.Exit`, but I don't know it is suitable.
https://talks.golang.org/2014/testing.slide#23

Reply:
we can try it.

Reply:
I think it works, but there is a problem that is shows the test coverage is 0% on the `os.Exit` tests because of it create a sub-process to execute the test. 
The main process cannot read the coverage from the sub-process.
I will search for another method to test it.

Reply:
I found the another way to test with os.Exit that enables to 100% coverage.
https://qiita.com/KEINOS/items/718c4237d8d0b3f4c26b#comment-2f55c169b593fb6733be
https://itnext.io/how-to-tdd-a-console-application-to-achieve-100-coverage-starting-from-main-test-go-934a617b080f

Reply:
It doesn't work as the `os.exit` call is not from our side but from the 3rd party dependencies.
We cannot refactor the implementation from them so we cannot achieve 100% coverage in this case.